x/gov/spec/02_state.md

```diff
@@ -11,24 +11,23 @@ be one active parameter set at any given time. If governance wants to change a
 parameter set, either to modify a value or add/remove a parameter field, a new
 parameter set has to be created and the previous one rendered inactive.
 
-```go
-type DepositParams struct {
-  MinDeposit        sdk.Coins  //  Minimum deposit for a proposal to enter voting period.
-  MaxDepositPeriod  time.Time  //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
+```proto
+message DepositParams {
+  repeated cosmos.base.v1beta1.Coin min_deposit = 1  //  Minimum deposit for a proposal to enter voting period.
```

Comment:
Is max_deposit missing here?

Also, I would recommend to use `+++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0-rc7/proto/cosmos/gov/v1beta1/gov.proto#L127-L145`, that will paste the exact snippet.